apps/openmw/mwworld/projectilemanager.cpp

```diff
@@ -92,7 +94,18 @@ namespace MWWorld
             attachTo = rotateNode;
         }
 
-        mResourceSystem->getSceneManager()->getInstance(model, attachTo);
+        osg::ref_ptr<osg::Node> ptr = mResourceSystem->getSceneManager()->getInstance(model, attachTo);
+
+        if (state.mIdMagic.size() > 1)
+            for (size_t iter = 1; iter != state.mIdMagic.size(); ++iter)
+            {
+                char numstr[8];
+                sprintf(numstr, "%d", (int)iter);
+                std::string node = "Dummy0";
```

Comment:
Or:

<pre>
std::ostringstream nodeName;
nodeName << "Dummy" << std::setw(2) << std::setfill('0') << iter;
</pre>

To handle the possibility of more than 9 magic effects.